android/src/ti/airship/AirshipTitaniumModule.java

```diff
@@ -100,6 +99,35 @@ public void setNamedUser(String namedUser) {
         UAirship.shared().getNamedUser().setId(namedUser);
     }
 
+    @Kroll.method
+    public void trackScreen(String screen) {
+        UAirship.shared().getAnalytics().trackScreen(screen);
+    }
+
+    @Kroll.method
+    @Kroll.setProperty
+    public void setIsDataCollectionEnabled(boolean enabled) {
+        UAirship.shared().setDataCollectionEnabled(enabled);
+    }
+
+    @Kroll.getProperty
+    @Kroll.method
+    public boolean getIsDataCollectionEnabled() {
```

Comment:
should these methods be named `setDataCollectionEnabled` and `getDataCollectionEnabled`? that would be more in line with our other JS ecosystem stuff.

Reply:
these are properties while the other ones are methods:
```
Airship.isDataCollectionEnabled = true
```

Reply:
oh, right